### PR TITLE
Ct 2356 prison number hint

### PR DESCRIPTION
**Pull request by digitalali-moj**
## Description
Add form hint text to prison number - now adding form hints to all places defined in en.yml - wrong object name was in the yml, fixed
 
## Self-review checklist

* [x] (1) Quick stakeholder demo done OR
* [x] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
 
### Screenshots
image

(See dev server, no hint texts were showing)
 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2358

### Deployment
n/a
 
### Manual testing instructions
Go through offender SAR journey and check form hitns

### CODE REVIEW

File: app/views/cases/offender_sar/_requested_info_step.html.slim [hunk 1]
```diff
@@ -8,8 +8,7 @@
 
 = form_for @case, url: case_create_action(@case.object), as: :offender_sar do |f|
 
-  = f.text_area :message, {rows: 12,
-    form_hint_text: t('helpers.hint.offender_sar_case_form.message')}
+  = f.text_area :message, {rows: 12}
```
Comment (mseedat-moj) [inline app/views/cases/offender_sar/_requested_info_step.html.slim]: Leave spaces between curly brace and contents i.e. `{ rows: 12 }`

File: config/locales/en.yml [hunk 2]
```diff
@@ -462,11 +462,11 @@ en:
...
-      offender_sar_case_form:
-        prison_number: "You can include multiple numbers separated by comma, space or paragraph return."
-        subject_aliases: "You can include multiple aliases separated by comma, space or paragraph return."
-        previous_case_numbers: "You can include multiple cases separated by comma, space or paragraph return."
-        other_subject_ids: "You can include multiple cases separated by comma, space or paragraph return."
+      offender_sar:
+        prison_number: "You can include multiple numbers separated by comma or space."
+        subject_aliases: "You can include multiple aliases separated by comma or space."
+        previous_case_numbers: "You can include multiple cases separated by comma or space."
+        other_subject_ids: "You can include multiple numbers separated by comma or space."
```
Comment (mseedat-moj) [inline config/locales/en.yml]: Ideally the keys should be in alphabetical order, and the quotes are not necessaryi.e.:
```
previous_case_numbers: You can include multiple cases separated by comma or space.
prison_number: You can include multiple numbers separated by comma or space.
other_subject_ids: You can include multiple numbers separated by comma or space.
subject_aliases: You can include multiple aliases separated by comma or space.  
```

You can usually re-arrange things alphabetically using your text editor e.g in sublime highlight text -> F5

**Review by mseedat-moj — CHANGES_REQUESTED** [review]
Looks fine to me - suggested changes (syntax)

File: app/views/cases/offender_sar/_requested_info_step.html.slim (hunk 1 above)
Comment (digitalali-moj) [reply app/views/cases/offender_sar/_requested_info_step.html.slim]: Yes I like that too, done. thanks

File: config/locales/en.yml (hunk 2 above)
Comment (digitalali-moj) [reply config/locales/en.yml]: Yes, F9 in the case of Linux, done as well. Thanks for picking these up, things I overlook all the time!

**Review by mseedat-moj — APPROVED** [review]

**Pull request merged**